webcomponents/search-wc/.browserslistrc

```diff
@@ -5,7 +5,7 @@
 # You can see what browsers were selected by your queries by running:
 #   npx browserslist
 
-> 0.5%
+> 5%
```

Comment:
what was the motivation for this btw? web component support?

Reply:
It's to avoid Angular to generate to many output files (ie maines5.js, maines2015.js etc..).